README.md

```diff
@@ -43,65 +43,67 @@ GPU: NVIDIA Corporation GV100GL [V100 PCIe 16GB]
 
 NVIDIA Driver version: 470.141.03
 
-Disk space: 50 GB required for the docker, 25 GB available free space in the docker container storage
+Disk space: 50 GB is required for the docker, 25 GB of available free space is needed in the docker container storage
 
 <h3> 🧩 Installation </h3>
 
-1. Set environment variable `PUBLIC_IP` to the ip address of the localhost.
+1. Set the environment variable `PUBLIC_IP` to the ip address of the localhost. This host must be reachable from where you will be running the browser. Otherwise, please use VNC to access the host.
+If running the application locally,  `PUBLIC_IP` can be set to `localhost`.
```

Comment:
Maybe we should say accessing via browser locally instead of running locally? Because they might go to cluster and think of run `launch.sh` via SSH which is running locally in that cluster but they then need to access via public ip of that cluster machine.